demo/run.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+function finish {
+  echo 'Removing demo environment'
+  echo '-----'
+  docker-compose down -v
+}
+trap finish EXIT
+
+function main() {
+  echo '-----'
+  echo '----------'
+  echo '---------------'
+  echo "Starting Ansible demo for Conjur"
+  echo '---------------'
+  echo '----------'
+  echo '-----'
+
+  createDemoEnvironment
+  waitForServer
+	createHostFactoryToken
```

Comment:
@orenbm not sure what's happening with this but it seems there are the random spacing inconsistencies through this file in particular.